lib/gooddata/lcm/actions/collect_users_brick_users.rb

```diff
@@ -15,9 +15,83 @@ class CollectUsersBrickUsers < BaseAction
         description 'Input source of the Users Brick. Needed to prevent ' \
                     'deletion of filters for a user that is to be removed.'
         param :users_brick_config, instance_of(Type::UsersBrickConfig), required: true
+
+        description 'Organization Name'
+        param :organization, instance_of(Type::StringType), required: false
+
+        description 'Domain'
+        param :domain, instance_of(Type::StringType), required: false
+
+        description 'Fail Early'
+        param :fail_early, instance_of(Type::BooleanType), required: false
+
+        description 'Strict'
+        param :strict, instance_of(Type::BooleanType), required: false
+
+        description 'Username'
+        param :username, instance_of(Type::StringType), required: false
+
+        description 'Password'
+        param :password, instance_of(Type::StringType), required: false
+
+        description 'Synchronization Mode (e.g. sync_one_project_based_on_pid)'
+        param :sync_mode, instance_of(Type::StringType), required: false, default: 'sync_domain_and_project'
+
+        description 'DataProduct to manage'
+        param :data_product, instance_of(Type::GdProductType), required: false
+
+        description 'Filters Config'
+        param :filters_config, instance_of(Type::HashType), required: false
+
+        description 'AWS Client'
+        param :aws_client, instance_of(Type::GdSmartHashType), required: false
+
+        description 'Input Source'
+        param :input_source, instance_of(Type::HashType), required: false
+
+        description 'Development Client Used for Connecting to GD'
+        param :development_client, instance_of(Type::GdClientType), required: false
+
+        description 'GDC Project'
+        param :gdc_project, instance_of(Type::GdProjectType), required: false
+
+        description 'GDC Project Id'
+        param :gdc_project_id, instance_of(Type::StringType), required: false
+
+        description 'GDC client protocol'
+        param :client_gdc_protocol, instance_of(Type::StringType), required: false
+
+        description 'GDC client hostname'
+        param :client_gdc_hostname, instance_of(Type::StringType), required: false
+
+        description 'GDC password'
+        param :gdc_password, instance_of(Type::StringType), required: false
+
+        description 'GDC username'
+        param :gdc_username, instance_of(Type::StringType), required: false
+
+        description 'DataLogger'
+        param :gdc_logger, instance_of(Type::GdLogger), required: false
+
+        description 'Client Used for Connecting to GD'
+        param :gdc_gd_client, instance_of(Type::GdClientType), required: false
+
+        description 'Segments filter'
+        param :segments_filter, array_of(instance_of(Type::StringType)), required: false
```

Comment:
some of these params are not really accessed in this action, why do they need to be specified?